apps/mitt-konto/src/Tracking.purs

```diff
@@ -3,10 +3,50 @@ module Tracking where
 import Effect (Effect)
 import Foreign (Foreign)
 import Prelude (Unit)
+import Effect.Uncurried (EffectFn2, EffectFn3, EffectFn5, runEffectFn2, runEffectFn3, runEffectFn5)
 
-foreign import data Tracker :: Type
-foreign import newTracker :: Effect Tracker
-foreign import pushPageLoad :: Tracker -> Effect Unit
+foreign import login_ :: EffectFn3 Cusno LoginMethod Result Unit
+foreign import reclamation_ :: EffectFn5 Cusno Subsno DateString Claim Result Unit
+foreign import tempAddressChange_ :: EffectFn5 Cusno Subsno StartDate EndDate Result Unit
+foreign import pauseSubscription_ :: EffectFn5 Cusno Subsno StartDate EndDate Result Unit
+foreign import unpauseSubscription_ :: EffectFn3 Cusno Subsno Result Unit
+foreign import deleteTempAddressChange_ :: EffectFn5 Cusno Subsno StartDate EndDate Result Unit
+foreign import changeName_ :: EffectFn2 Cusno Result Unit
+foreign import changeAddress_ :: EffectFn2 Cusno Result Unit
+
+type Cusno = String
+type Subsno = Int
+type DateString = String
+type Claim = String
+type StartDate = String
+type EndDate = String
+type Result = String
+type LoginMethod = String
+  
+-- Int = subsno, String = date
+reclamation :: Cusno -> Subsno -> DateString -> Claim -> Result -> Effect Unit
+reclamation = runEffectFn5 reclamation_
+
+tempAddressChange :: Cusno -> Subsno -> StartDate -> EndDate -> Result -> Effect Unit
```

Comment:
Why not do the date formatting in the functions that take the dates? Then we'd have some type safety (not just aliases). Here I could just pass whatever strings I wanted even though they're supposed to be dates.